.github/workflows/docs.yml

```diff
@@ -56,6 +56,7 @@ jobs:
         run: python -m pip install nox docutils
 
       - name: build example
+        timeout-minutes: 10
```

Comment:
I'd make this 16' so it doesn't fail current examples ^_^

Reply:
I did 12min, pet-mad-nc takes 10min currently, so this should be enough of a buffer for run to run variations